### ISSUE TITLE
Cannot find notifySplices

### ISSUE THREAD

**Issue by kwaclaw** [nippur72/PolymerTS#75]
Using version 0.1.24.
notifySplices is needed for array observation when one cannot use Polymer's array mutation functions.
(notifyPath does not work properly for arrays).

**Comment by nippur72** [comment]
Ok thanks, I've addded it to the [0.1.25](https://github.com/nippur72/PolymerTS/releases/tag/0.1.25).

If my understanding of the docs is correct, this should be the typescript signature:
```ts
notifySplices(path: string, splices: {
            index: number;
            removed: Array<any>;
            addedCount: number;
            object: Array<any>;
            type: "splice";
        }): void;
```

**Comment by kwaclaw** [comment]
Sorry, I did not check properly. splices should be an array of these "splice" objects. Without defining a new type, it should be:

        notifySplices(path: string, splices: {
            index: number;
            removed: Array<any>;
            addedCount: number;
            object: Array<any>;
            type: "splice";
        }[]): void;

**Comment by nippur72** [comment]
oops! 

Thanks, it's fixed in 0.1.26.
